grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -520,4 +520,30 @@ class UserService {
         }
         user.save(failOnError: true)
     }
+
+    User createOrUpdateUser(Map params) {
+        User user = User.findByUsername(params.username)
+        if (!user) {
+            user = new User(params)
+            user.active = true
+            user.password = "password"
```

Comment:
do we really set the password like that?

Reply:
For imported users, yes. I think the reason was due to the fact that we added this feature as part of an implementation so we didn't need to create users manually. The user that was doing the importing had a process to follow with respect to communicating with users and having them reset the password. 

This should have been accompanied with a feature that required the user to reset their password if it was set to this default value. However, that part never happened. 

I just commented about this tech debt in a ticket https://pihemr.atlassian.net/browse/OBPIH-5531?focusedCommentId=147513